### PR TITLE
[v2.5.8] final rc

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -15,7 +15,7 @@ ENV HELM_VERSION v3.3.1
 ENV KUSTOMIZE_VERSION v3.5.4
```
Comment (cbron) [inline Dockerfile.dapper:15]: RKE is missing: https://github.com/rancher/rancher/blob/release/v2.5/go.mod#L102
Make sure whole release checklist step 4 is done.

Comment (cbron) [reply Dockerfile.dapper:15]: Looks like KDM stuff missing too: https://github.com/rancher/rancher/blob/release/v2.5/pkg/settings/setting.go#L225

**Review by cbron — CHANGES_REQUESTED** [review]

**Review by cbron — COMMENTED** [review]
lgtm, waiting for QA

File: pkg/apis/go.mod [hunk 2]
```diff
@@ -10,7 +10,7 @@ replace (
 require (
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.1-rc14
+	github.com/rancher/gke-operator v1.0.1
 	github.com/rancher/norman v0.0.0-20210225010917-c7fd1e24145b
 	github.com/rancher/rke v1.2.8-rc4
```
Comment (kinarashah) [inline pkg/apis/go.mod:15]: could you update rke here as well

**Review by kinarashah — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -90,7 +90,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.1-rc14
+	github.com/rancher/gke-operator v1.0.1
```
Comment (cbron) [inline go.mod:93]: The commit looks weird now, and this is already done: https://github.com/rancher/rancher/blob/release/v2.5/go.mod#L93 
Need to rebase I'm guessing

File: scripts/package [hunk 4]
```diff
@@ -7,9 +7,9 @@ source $(dirname $0)/version
 
 ARCH=${ARCH:-"amd64"}
 SYSTEM_CHART_REPO_DIR=build/system-charts
-SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH:-"dev-v2.5"}
+SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH:-"release-v2.5"}
 CHART_REPO_DIR=build/charts
-CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH:-"dev-v2.5"}
+CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH:-"release-v2.5"}
 
 cd $(dirname $0)/../package
```
Comment (cbron) [inline scripts/package:15]: CI error was weird, looks like UI tag might be messed up

**Comment by cbron** [comment]
New PR: https://github.com/rancher/rancher/pull/32565

**Pull request closed without merging**